### PR TITLE
Issue-49 | Created index.html file to represent carousel component

### PR DESCRIPTION
**Pull request by iScous**
In progress: created index.html file for carousel component. Made a functionality that allows to switch between pictures. Fixed with the latest comments. Added animation, bullets. Some code updates. Functionality for bullets is not complete yet.
Issue: #49

### CODE REVIEW

File: src/carousel/slider.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        const sliderCotainerWrapper = document.querySelector('#sliderContainer-wrapper');
+
+        for (let i = 0; i < this.images.length - 2; i++) {
+            const sliderDiv = document.createElement('div');
+            sliderDiv.className = 'showSlide';
+            const sliderImg = document.createElement('img');
+            sliderImg.src = this.images[i].src;
+            sliderDiv.appendChild(sliderImg);
+            sliderCotainerWrapper.appendChild(sliderDiv);
+        }
```
Comment (arsenykonohovwork) [inline src/carousel/slider.js]: @iScous still using here not appropriate loops; Use Array like methods or Object methods for enumerated data structures;

**Comment by arsenykonohovwork** [comment]
@iScous sorry i didn't request changes recently (((

**Comment by arsenykonohovwork** [comment]
@iScous I've composed them, but didn't send them all;
Accept my heartfelt apologies;

**Comment by iScous** [comment]
@arsenykonohovwork working on options for the component

File: src/carousel/slider.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+class Carousel {
+    constructor(firtSlideToShow, slidesToShow, slidesToScroll, speedOfScrolling, autoplay, autoplaySpeed, infinite) {
+        this.firtSlideToShow = firtSlideToShow;
+        this.slidesToShow = slidesToShow;
+        this.slidesToScroll = slidesToScroll;
+        this.speedOfScrolling = speedOfScrolling;
+        this.autoplay = autoplay;
+        this.autoplaySpeed = autoplaySpeed;
+        this.infinite = infinite;
+    }
```
[the remark this answers is not in the document]
Comment (iScous) [reply src/carousel/slider.js]: fixed

File: src/carousel/slider.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            this.firtSlideToShow = 1
+        }
+
+        if (this.firtSlideToShow < 1) {
+            this.firtSlideToShow = slides.length
+        }
+
+        for (let i = 0; i < slides.length; i++) {
+            slides[i].style.display = "none";
+        }
```
[the remark this answers is not in the document]
Comment (iScous) [reply src/carousel/slider.js]: fixed

File: src/carousel/slider.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        const slides = document.getElementsByClassName("showSlide");
+
+        if (this.firtSlideToShow > slides.length) {
+            this.firtSlideToShow = 1
+        }
+
+        if (this.firtSlideToShow < 1) {
+            this.firtSlideToShow = slides.length
+        }
+
```
[the remark this answers is not in the document]
Comment (iScous) [reply src/carousel/slider.js]: fixed

File: src/carousel/slider.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        carousel.displaySlides(this.firtSlideToShow += next);
+    }
+}
+
+const carousel = new Carousel(1);
+carousel.displaySlides();
+
+function nextSlide(next) {
+    carousel.nextSlide(next);
+}
```
[the remark this answers is not in the document]
Comment (iScous) [reply src/carousel/slider.js]: fixed

File: src/carousel/slider.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    nextSlide(next) {
+        carousel.displaySlides(this.firtSlideToShow += next);
+    }
+}
+
+const carousel = new Carousel(1);
+carousel.displaySlides();
+
```
[the remark this answers is not in the document]
Comment (iScous) [reply src/carousel/slider.js]: in progress

File: src/carousel/slider.js (hunk 1 above)
Comment (iScous) [reply src/carousel/slider.js]: fixed

**Pull request merged**